src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -857,7 +852,12 @@ protected CountingVariantFilter makeVariantFilter() {
             headerLines.add(GATKVCFHeaderLines.getInfoLine(GATKVCFConstants.ORIGINAL_DP_KEY));
         }
 
-        headerLines.addAll(Arrays.asList(ChromosomeCounts.descriptions));
+        for (final String key : ChromosomeCounts.keyNames) {
+            headerLines.removeIf(line->line instanceof VCFInfoHeaderLine && ((VCFInfoHeaderLine)line).getID().equals(VCFConstants.DEPTH_KEY));
```

Comment:
Actually, maybe this line SHOULD reference `key` instead of `VCFConstants.DEPTH_KEY`. Then it would make more sense ?

Reply:
Can you add a comment here saying this loop/removal is necessary due to the duplicate line bug in htsjdk.

Reply:
You're absolutely right, it should be `key` instead of `VCFConstants.DEPTH_KEY`, this was likely a copy-paste mistake, thank you. This loops through the `AC`, `AF`, and `AN` tags, removes them if they were present and overwrites them with the respective standard GATK header lines. Subsequently, this is done for the `DP` tag below the loop.